frontend/src/components/organisms/TournamentsCardsGallery/TournamentsCardsGallery.tsx

```diff
@@ -0,0 +1,127 @@
+import {useEffect, useState} from "react";
+import {TournamentRegistrationsRestApi} from "../../../restapi/tournament-registrations";
+import {TournamentDetailsRestApi} from "../../../restapi/tournament-details/TournamentDetailsRestApi";
+import {TournamentDetailsDto} from "../../../restapi/tournament-details/TournamentDetailsDto";
+import {TournamentRegistrationsWithDetailsDto} from "../../../restapi/tournament-registrations/TournamentRegistrationsWithDetailsDto";
+
+
+const loadedTournamentsCards = [
+    {
+        tournamentId: '1',
+        tournamentName: 'Turniej z jajem',
+        status: 'OPENED',
+        registeredPlayersIds: [],
+    },
+    {
+        tournamentId: '2',
+        tournamentName: 'Turniej bez jaja',
+        status: 'CLOSED',
+        registeredPlayersIds: [],
+    },
+    {
+        tournamentId: '3',
+        tournamentName: 'Turniej wrocławski',
+        status: 'OPENED',
+        registeredPlayersIds: [],
+    },
+    {
+        tournamentId: '4',
+        tournamentName: 'Turniej kaliski',
+        status: 'CLOSED',
+        registeredPlayersIds: [],
+    },
+]
+
+const tournamentsRegistrationsDetailsList = [
+    {
+        tournamentId: '2480fc2c-bbd7-427c-9439-02f76583aef5',
+        tournamentName: 'Taka na sztywno',
+    },
+    {
+        tournamentId: '2480fc2c-bbd7-427c-9439-02f76583aef6',
+        tournamentName: 'Taka na luźno',
+    },
+]
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+            flexWrap: 'wrap',
+            justifyContent: 'space-around',
+        },
+        gridList: {
+            flexWrap: 'nowrap',
+            // Promote the list into his own layer on Chrome. This cost memory but helps keeping high FPS.
+            transform: 'translateZ(0)',
+            '&::-webkit-scrollbar-track':
+                {
+                    webkitBoxShadow: 'inset 0 0 6px rgba(0,0,0,0.3)',
+                    borderRadius: '4px',
+                    backgroundColor: '#646464',
+                },
+            '&::-webkit-scrollbar': {
+                height: '10px',
+            },
+            '&::-webkit-scrollbar-thumb': {
+                borderRadius: '4px',
+                webkitBoxShadow: 'inset 0 0 6px rgba(0,0,0,.3)',
+                backgroundColor: theme.palette.primary.main,
+            }
+        },
+    }),
+);
+
+const StyledWrapper = styled("div")({
+    boxSizing: "border-box",
+    margin: 0,
+    padding: 0,
+    maxWidth: MIN_CARD_COMPONENT_WIDTH,
+    width: MIN_CARD_COMPONENT_WIDTH,
+})
+
+const TournamentCardsGallery = () => {
+    const [tournamentsRegistrations, setTournamentsRegistrations] = useState<TournamentRegistrationsWithDetailsDto[]>(loadedTournamentsCards); // usunąć initial state na produkcję
+    const [tournamentsDetails, setTournamentsDetails] = useState<TournamentDetailsDto[]>(tournamentsRegistrationsDetailsList); // usunąć initial state na produkcję
```

Comment:
Dodaj w komentarzach `TODO:`
ESLint wyłapuje frazę `// TODO:` i pomagają w wyświetlaniu i inforowaniu o tym.
Jak zostawisz tak jak jest to zapomnisz. A jak będzie `// TODO: coś tam` to się wyświetli!
No i dodatkowo edytory kodu pozwalają szybko wyszukać TODO - nawet są specjalne wtyczki do VS Code, które tworzą drzewa do ` // TODO:`